src/model.ts

```diff
@@ -118,11 +118,18 @@ export const applyModelConfig = <T extends typeof JSORMBase>(
   let k: keyof ModelConfigurationOptions
 
   for (k in config) {
-    if (config.hasOwnProperty(k)) {
+    if (config.hasOwnProperty(k) && k !== 'jwt') {
```

Comment:
I think I'm fine with this with a comment. Alternatively, I wonder if it might make more sense to manually do the JWT setup on line 119, then end the function with iterating props and assigning them. This way the logic isn't broken up between line 121 and 130, might read a bit more sensibly.

Reply:
> do the JWT setup on line 119

Do you mean dealing specifically with all of the jwt-related ones and then finishing up with everything else in the config?

Reply:
Yes